common/include/window.h

```diff
@@ -50,6 +50,79 @@ static inline void set_embedded_window_pointer(embed_window_pointer_t *wp, windo
 
 static inline void set_embedded_window_pointer(ignore_window_pointer_t *, window *) {}
 
+struct window
```

Comment:
I see only one forward declaration as `struct window`, in `fwd-window.h`:
```
$ git grep 'struct window;'
common/include/fwd-window.h:21:struct window;
```
The forward-declaration header is included by some other files, such as `mouse.h`, but the declaration itself is not repeated.
```
$ git grep -F '"fwd-window.h"'
common/include/editor/editor.h:35:#include "fwd-window.h"
common/include/editor/info.h:29:#include "fwd-window.h"
common/include/mouse.h:20:#include "fwd-window.h"
common/include/window.h:23:#include "fwd-window.h"
common/main/automap.h:53:#include "fwd-window.h"
common/main/game.h:37:#include "fwd-window.h"
common/main/newmenu.h:39:#include "fwd-window.h"
```